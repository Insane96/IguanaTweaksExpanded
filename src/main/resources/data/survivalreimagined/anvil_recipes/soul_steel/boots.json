{
  "left": {
    "item": "minecraft:iron_boots"
  },
  "right": {
    "item": "survivalreimagined:soul_steel_ingot"
  },
  "amount": 4,
  "result": {
    "item": "survivalreimagined:soul_steel_boots"
  },
  "keep_durability": true,
  "chance_to_break": 0.075
}