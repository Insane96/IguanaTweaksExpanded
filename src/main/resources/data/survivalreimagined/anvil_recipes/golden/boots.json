{
	"item_to_repair": {
		"id": "minecraft:golden_boots"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:golden_ingot"
			},
			"amount": 4
		},
		{
			"repair_material": {
				"id": "minecraft:golden_nugget"
			},
			"amount": 36
		}
	]
}